release/rke2/rke2.go

```diff
@@ -35,23 +36,36 @@ func ImageBuildBaseRelease(ctx context.Context, ghClient *github.Client) error {
 			logrus.Info("version " + version.Version + " is not stable")
 			continue
 		}
-		v := "v" + strings.Split(version.Version, "go")[1] + "b1"
-		logrus.Info("stripped version: " + v)
-		_, _, err := ghClient.Repositories.GetReleaseByTag(ctx, "rancher", imageBuildBaseRepo, v)
+		version := strings.Split(version.Version, "go")[1]
+		alpineTag := version + "-alpine" + alpineVersion
+
+		err := docker.CheckImageArchs(ctx, "library", "golang", alpineTag, []string{"amd64", "arm64", "s390x"})
```

Comment:
Can you scope lock this err value?

Reply:
Can you explain it?

Reply:
Generally, single return err values should be done like this: 

if err := ...

This locks this err value to the if scope.